src/clipboard.js

```diff
@@ -95,7 +95,7 @@ export default class Clipboard extends Plugin {
 	 * @inheritDoc
 	 */
 	static get pluginName() {
-		return 'clipboard/clipboard';
```

Comment:
See documentation of `Plugin.pluginName`. It should be `clipboard/clipboard` – one is the package name (meaningful part of it) and the other is the module inside it.